src/main/java/org/opensearch/ad/task/ADTaskManager.java

```diff
@@ -2212,4 +2254,91 @@ private void resetAllRealtimeTasksAsStopped() {
             );
     }
 
+    public String convertEntityToString(ADTask adTask) {
```

Comment:
Any reason for not creating some of these methods in the `Entity` class's set of methods? I remember Kaituo has some for other inputs to convert to Entity there, like [fromJsonArray](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/model/Entity.java#L388)

Reply:
This logic is only special to historical task, not general enough for other usage. If somewhere else need to reused, we can move this to Entity. Currently prefer to keep these in task manager to not confuse others.